output/reduced-cost-mod-mappings.ts

```diff
@@ -32,6 +31,7 @@ export const normalToReducedMod: { [normalModHash: number]: number } = {
   '1130820873': 3461249873,
   '1180408010': 2568808786,
   '1193713026': 1672155562,
+  '1208761894': 84503918,
```

Comment:
@robojumper

Reply:
Thank you. Bungie reduced the cost of the regular Empowered Finish from 3 to 1 (missing in patch notes btw). Guess I should've expected Bungie to not obliterate the cheap copy.